### PR TITLE
Add Annual Report page

### PR DESCRIPTION
**Pull request by fontofonto**

### CODE REVIEW

**Comment by fontofonto** [comment]
@chochihim @mailtime-wing may need you help to merge the branch

File: package.json [hunk 1]
```diff
@@ -53,15 +54,15 @@
...
+    "sharp": "0.29.3"
   },
   "keywords": [
     "gatsby"
   ],
   "license": "MIT",
   "scripts": {
     "build": "gatsby build",
-    "develop": "gatsby develop",
+    "develop": "gatsby develop --host=0.0.0.0",
```
Comment (mailtime-wing) [inline package.json:65]: what is this for?

File: package.json [hunk 2]
```diff
@@ -87,7 +88,6 @@
     ],
     "*.{js,jsx}": [
       "prettier --write",
-      "eslint --max-warnings=0 --ignore-pattern '!.eslintrc.js' --fix",
```
Comment (mailtime-wing) [inline package.json:90]: why remove the eslint checking here?

File: package.json (hunk 1 above)
Comment (fontofonto) [reply package.json:65]: Make it easier for me to check and test on mobile device on the local network

File: package.json (hunk 2 above)
Comment (fontofonto) [reply package.json]: added back

**Pull request merged**